.github/workflows/secretlint-only-on-changed-files.yml

```diff
@@ -0,0 +1,23 @@
+name: Secretlint only on changed files
+on: [push, pull_request]
+
+# secretlint + git diff on Pull Request
+# https://github.com/secretlint/secretlint
+
+jobs:
+  run-secretlint:
+    name: "Secretlint only on changed files"
+    runs-on: ubuntu-18.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - uses: technote-space/get-diff-action@v6
+      - name: setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install
+        run: npm install
+      - name: Lint with Secretlint
+        run: npx --no-install secretlint ${{ env.GIT_DIFF_FILTERED }}
```

Comment:
This workflow's output is as bellows;
https://github.com/gkzz/secretlint-github-actions-example/actions/runs/2043608456